pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.poetry]
+name = "trackma"
+version = "2022.10"
+description = "Open multi-site list manager"
+authors = ["z411 <z411@krutt.org>"]
+license = "GPL-3.0-or-later"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "3.7.14"
```

Comment:
This should definitely be `^3.7`. pycairo should be an optional dependency, but [3.6 has reached EOL by almost a year now](https://peps.python.org/pep-0494/) so requiring 3.7 should be fine. That also allows us to use `from __future__ import annotations` and makes way for a lot of useful annotation features that have been added in later versions.